### DISCUSSION TITLE
css object-fit attribute for website thumbnail (.card-img) listings?

### DISCUSSION THREAD

**Discussion by ohshane** [quarto-dev/quarto-cli#10492] [Q&A] [answered]
### Description

is it possible to add a css `object-fit` attribute for website card-img thumbnails?

Sometimes there are some images which is horizontally long (such as logos) which gets trimmed because of the default behavior of `object-fit: cover`.

Is there a way to set a per-post `object-fit` attributes such as `cover`, `contain`, `fill` etc?

**Comment by mcanouil** [comment, marked as answer]
Overwrite the CSS via the `css` key or use `include-*` to include your CSS as plain text.

You can also use JavaScript.

**Reply by ohshane** [reply]
I'm pretty new to Quarto. Many thanks!
